site/server/test/UserProgressRepositorySpec.scala

```diff
@@ -0,0 +1,105 @@
+import com.fortysevendeg.exercises.persistence.repositories.UserProgressDoobieRepository.{ instance ⇒ repository }
+import doobie.imports._
+import org.scalatest._
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+import test.database.TestDatabase
+import org.scalacheck.Shapeless._
+
+import scalaz.concurrent.Task
+
+class UserProgressRepositorySpec
+    extends PropSpec
+    with GeneratorDrivenPropertyChecks
+    with Matchers
+    with ArbitraryInstances {
+
+  val db = TestDatabase.create
```

Comment:
Why not using the other db instance that is used in other tests?

On Mon, Feb 15, 2016, 8:01 PM Juan Pedro Moreno <notifications@github.com>
wrote:

> In site/server/test/UserProgressRepositorySpec.scala
> <https://github.com/47deg/scala-exercises/pull/205#discussion_r52958237>:
>
> > +import com.fortysevendeg.exercises.persistence.repositories.UserProgressDoobieRepository.{ instance ⇒ repository }
> > +import doobie.imports._
> > +import org.scalatest._
> > +import org.scalatest.prop.GeneratorDrivenPropertyChecks
> > +import test.database.TestDatabase
> > +import org.scalacheck.Shapeless._
> > +
> > +import scalaz.concurrent.Task
> > +
> > +class UserProgressRepositorySpec
> > +    extends PropSpec
> > +    with GeneratorDrivenPropertyChecks
> > +    with Matchers
> > +    with ArbitraryInstances {
> > +
> > +  val db = TestDatabase.create
>
> Yep, this factory method creates an instance for this test class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/47deg/scala-exercises/pull/205/files#r52958237>.
>